apps/payments/next/project.json

```diff
@@ -79,5 +79,5 @@
       "command": "pm2 delete apps/payments/next/pm2.config.js"
     }
   },
-  "tags": []
+  "tags": ["lib", "payments"]
```

Comment:
Isn't this an app?

Reply:
Sorry, I merged before seeing this. This is definitely noted. I'll address it shortly. For now, the CI should still function properly even though the semantics of this tag is off.